sbt-version-policy/src/main/scala/sbtversionpolicy/internal/DependencyCheck.scala

```diff
@@ -21,7 +21,12 @@ object DependencyCheck {
       .modules
       .filter(!_.evicted)
       .map(_.module)
-      .filter(module => !excludedModules.contains(module.organization -> module.name))
+      .filter {
+        case module if module.extraAttributes.contains("sbtVersion") && module.extraAttributes.contains("scalaVersion") =>
+          !excludedModules.contains(module.organization -> s"${module.name}_${module.extraAttributes("scalaVersion")}")
```

Comment:
I wonder if we should also append the `sbtVersion`?

```suggestion
          !excludedModules.contains(module.organization -> s"${module.name}_${module.extraAttributes("scalaVersion")}_${module.extraAttributes("sbtVersion")}")
```

Would it be possible to add a test for this? In `sbt-version-policy/src/sbt-test/sbt-version-policy`, add a directory `sbt-plugin-submodule` with your test. I can help you if you would like more help :)

Reply:
Sure! No problem.

Reply:
Hey @julienrf, I've added a test [here](https://github.com/scalacenter/sbt-version-policy/pull/199/commits/818d333915dbb9f0153843e032cddc23cb3f2f51). What is the motivation for appending the `sbtVersion` value? I don't think something like `name_2.13_1.0` will ever be present in the `excludedModules` list.

Reply:
Hey @alejandrohdezma. Thank you for the update. After carefully checking I know see that you are right and we only need to take into account the Scala binary version, not the sbt version.

I’ve pushed a commit to your branch to move the piece of logic that handles sbt plugins to the same place as the other types of modules (where we compute the set of excluded modules).

I’ve also created a more specific test because the `example-sbt-dynver` should be a canonical example of how to use sbt-version-policy with sbt-dynver.

Reply:
Great! That makes total sense! Thank you!

Reply:
Thank _you_! Your fix should be available in release 3.2.1.